api/v1alpha1/dexserver_types.go

```diff
@@ -141,17 +141,17 @@ type ClaimMappingSpec struct {
 	// preferredUsername is the list of claims whose values should be used as the preferred username.
```

Comment:
Can you update the 3 comments here to explain that while there might be a list on the oidc provider, we are only going to support one entry or something like that?